pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java

```diff
@@ -366,8 +380,11 @@ public void healthcheck() throws PulsarAdminException {
     }
 
     @Override
-    public CompletableFuture<Void> healthcheckAsync() {
+    public CompletableFuture<Void> healthcheckAsync(TopicVersion topicVersion) {
         WebTarget path = adminBrokers.path("health");
+        if (Objects.nonNull(topicVersion)) {
```

Comment:
any reason by just not checking `topicVersion != null`, and avoiding extra method call.

Reply:
No reason can change to using the operator.

Reply:
done